app/views/users/new.html.erb

```diff
@@ -1,33 +1,48 @@
+<% provide(:title, 'Sign up') %>
+<!DOCTYPE html>
```

Comment:
- [ ] We don't need any `doctype`, `html`, `head` tags in here. Any view file when gets rendered will replace the `<=% yield %>` in `application.html.erb`. All this code will be removed by the browser.